src/Components/Filter/Filter.jsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { useFilter } from "../../context/filterContext";
+import "./Filter.css";
+
+const Filter = () => {
+  const {
+    state: {
+      sortBy,
+      category: { Fiction, NonFiction, SelfHelp },
+      price,
+      BestSeller
+    },
+    dispatch,
+  } = useFilter();
+  return (
+    <div className="filters">
+      <div className="filter-heading">
+        <h3 className="filter-head main">Filters</h3>
+        <p onClick={()=>dispatch({type:"CLEAR"})} className="filter-head clear">Clear All</p>
+      </div>
+      <form className="filter-form">
+        {/* SORT BY PRICE AND NAME  */}
+        <div className="filter-category --Sortby">
+          <h3 className="filter-head">Sort By</h3>
+          <div>
+            <label htmlFor="lowtohigh">
+              <input
+                type="radio"
+                id="lowtohigh"
+                name="sortBy"
+                value="lowtohigh"
+                checked={sortBy === "LOW_TO_HIGH"}
+                onChange={() => dispatch({ type: "LOW_TO_HIGH" })}
+              />
+              Price (Low-High)
+            </label>
+          </div>
+          <div>
+            <label htmlFor="hightolow">
+              <input
+                type="radio"
+                id="hightolow"
+                name="sortBy"
+                value="hightolow"
+                checked={sortBy === "HIGH_TO_LOW"}
+                onChange={() => dispatch({ type: "HIGH_TO_LOW" })}
+              />
+             Price (High-Low)
+            </label>
+          </div>
+          <div>
+            <label htmlFor="nameA-Z">
+              <input
+                type="radio"
+                id="nameA-Z"
+                name="sortBy"
+                value="nameA-Z"
+                checked={sortBy === "NAMEA-Z"}
+                onChange={() => dispatch({ type: "NAMEA-Z" })}
+              />
+              Book Title (A-Z)
+            </label>
+          </div>
+          <div>
+            <label htmlFor="nameZ-A">
+              <input
+                type="radio"
+                id="nameZ-A"
+                name="sortBy"
+                value="nameZ-A"
+                checked={sortBy === "NAMEZ-A"}
+                onChange={() => dispatch({ type: "NAMEZ-A" })}
+              />
+              Book Title (Z-A)
+            </label>
+          </div>
+          <div>
+            <label htmlFor="rating">
+              <input
+                type="radio"
+                id="rating"
+                name="sortBy"
+                value="rating"
+                checked={sortBy === "RATING"}
+                onChange={() => dispatch({ type: "RATING" })}
+              />
+               Rating (High-Low)
+            </label>
+          </div>
+        </div>
+
+        <div className="filter-category --bestSeller">
```

Comment:
You can keep the name of the class as bestSeller. Was there any specific reason to give --bestSeller?